jesus-king-tech/src/main/java/com/api/jesus_king_tech/alimento/AlimentoController.java

```diff
@@ -0,0 +1,93 @@
+package com.api.jesus_king_tech.alimento;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/alimentos")
+
+public class AlimentoController {
+
+    @Autowired
+    AlimentoRepository alimentoRepository;
+
+
+
+    @GetMapping
+    public ResponseEntity<List<Alimento>> listarAlimentos() {
+        List<Alimento> alimentos = alimentoRepository.findAll();
+        if (alimentos.isEmpty()) {
+            return ResponseEntity.noContent().build();
+        }
+        return ResponseEntity.status(200).body(alimentos);
+    }
+
+
+
+    @PostMapping
+    public ResponseEntity<Alimento> cadastrarAlimento(@RequestBody Alimento alimento) {
+
+        Optional<Alimento> existenteOpt = alimentoRepository.findByNome(alimento.getNome());
+
+        if (existenteOpt.isPresent()) {
+            Alimento existente = existenteOpt.get();
+            existente.setQuantidade(existente.getQuantidade() + 1);
```

Comment:
Aqui, vc precisa pegar essa "quantidade" de alguma forma do front end, eu imaginei da seguinte forma

o front manda uma req :

{
nome: arroz,
peso: 5.0,
quantidade: 2
}

Com esse body, vc vai pegar o quantidade e fazer um for para fazer a insersão duas vezes no banco... 
E só vai inserir no banco as outras informações sem ser a quantidade, deu pra entender ??